### DISCUSSION TITLE
Is it possible to incorporate an ssl certificate to get rid of the security warning?

### DISCUSSION THREAD

**Discussion by correii** [DSJAS/DSJAS#42] [Ideas]
All modern browsers give me a "not secure" warning when on the site, because it's not https. I've looked and it doesn't seem like there's a way around this without adding an ssl certificate to the site so you can manually trust it. If there is an alternative please let me know, I just need to get rid of the warning so scammers don't get suspicious. I can't imagine a real bank using an unsecured connection.

**Comment by ejv2** [comment]
Try looking into self-signed certificates and how to add them to your trust store. Most operating systems provide a way to do this, and the OpenSSL CLI tool has a utility to generate a self-signed certificate.

**Comment by correii** [comment]
Lit, thanks man I’ll try to figure that out
